contrib/digitalocean/create

```diff
@@ -2,7 +2,7 @@
 
 set -eo pipefail -o nounset
 
-terraform apply
+terraform apply | grep --line-buffered -v user_data
```

Comment:
Note that this will prevent the user_data contents from dumping out on the screen.